.pre-commit-config.yaml

```diff
@@ -5,6 +5,7 @@ repos:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+    -   id: pytest
```

Comment:
It's debatable but usually you wouldn't want to put the unit tests in pre-commit. You want this to be quick and sometimes tests can take a while.

Reply:
second that thought, might be worth creating a CI workflow using Github Actions to run tests on commit or some other condition. Could create a new issue for that

Reply:
sounds fair, removed